couchdb/10-docker-default.ini

```diff
@@ -18,6 +18,15 @@ bind_address = 0.0.0.0
 server_options = [{recbuf, 262144}]
 socket_options = [{sndbuf, 262144}, {nodelay, true}]
 require_valid_user = true
+authentication_handlers = {chttpd_auth, cookie_authentication_handler}, {chttpd_auth, proxy_authentication_handler}, {chttpd_auth, default_authentication_handler}
+
+[chttpd_auth]
+require_valid_user = true
```

Comment:
I don't think we need to set this here. It is already set on line `20` (and I think it belongs in the `chttpd` section antyway).
```suggestion
```